packages/grafana-ui/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { cloneElement, FC, HTMLAttributes, ReactElement, ReactNode } from 'react';
+import { css, cx } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme, styleMixins } from '../../themes';
+import { Tooltip, PopoverContent } from '../Tooltip/Tooltip';
+import { OnTagClick } from '../Tags/Tag';
+import { TagList } from '../Tags/TagList';
+
+interface ContainerProps extends HTMLAttributes<HTMLOrSVGElement> {
+  /** Customise the container html element for the card. Defaults to div */
+  tag?: keyof JSX.IntrinsicElements;
```

Comment:
I'm not convinced about this customization. IMO Card should always be div, and additional behaviours / layout should be transparent to card. By this I especially mean that we should do:

```tsx
<ul>
  <li><Card .... /></li>
  ...
</ul>
```
Rather than:
```tsx
<ul>
  <Card tag="li" .... />
  ...
</ul>
```

It's easier to reason about and removes the unnecessary "complexity" of the CardContainer component.

@jackw WDYT?

Reply:
The argument makes sense but at the same time I'm not a huge fan of this added layout complexity in case we need to render card as a list element.

Reply:
Yeah, but with the `tag` property you are introducing a new pattern to the API. I cannot recall any situation when we allow DOM customization this way and we always delegate the layout responsibility to the outer component rather than encapsulating it within a component. 

I suggest dropping this property and plain div always for the card.

Reply:
> delegate the layout responsibility to the outer component rather than encapsulating it within a component
I'm a little late to the table here but I just wanted to say I agree with this approach. The less it knows the better. If the card needs to be rendered in a list, I'd expect a List or ListItem component to handle that piece of layout.

Reply:
agree with Dominik that controlling what tag is used feels unnecessary and not something we do with any other component